api/action/html-to-pdf.go

```diff
@@ -6,7 +6,8 @@ import (
 	"github.com/opsway/documents/cmd/document"
 )
 
-func HtmlToPdfGet(w http.ResponseWriter, r *http.Request) {
+// HTMLToPdfGet renders pdf from html
+func HTMLToPdfGet(w http.ResponseWriter, r *http.Request) {
```

Comment:
@u5surf  Please do not rename. There is a certain consistency in the names at the moment, for example `Html`, `Pdf`, `Url`.  If you use style `HTML`, then for consistency you need to do it everywhere `PDF`, `ULR`, for example `HTMLToPDFGet`, but I see no reason to do it.

Reply:
I additionally consulted with colleagues and we came to the conclusion to use abbreviations
, for example `HTML`, `PDF`, `URL`.

Reply:
use `HTMLToPDFGet`